src/loader.js

```diff
@@ -115,6 +118,39 @@ function setupModuleLoader(window) {
           _runBlocks: runBlocks,
 
           /**
+           * @ngdoc method
+           * @name angular.Module#info
+           * @module ng
+           *
+           * @param {Object=} info Information about the module
+           * @returns {Object|Module} The current info object for this module if called as a getter,
+           *                          or `this` if called as a setter.
+           *
+           * @description
+           * Additional info about this module
+           * For example you could put the version of the module in here.
+           *
+           * ```js
+           * angular.module('myModule', []).info({ version: '1.0.0' });
+           * ```
+           *
+           * You can retrieve this information during runtime via the
+           * {@link $injector#modules `$injector.modules`} property:
+           *
+           * ```js
+           * var version = $injector.modules['myModule'].info().version;
```

Comment:
Yes, I am referring to the `providerInjector` that can be injected into providers/config blocks. E.g.:

```js
.provider('foo', ['$injector', function fooProvider(providerInjector) {
  this.$get = ['$injector', function foo(instanceInjector) {
    console.log(providerInjector === instanceInjector);   // false
  }
})
```

(Again, I don't expect many people to be actually using it, but you never know :smiley:)

--
Regarding `angular.module('someModule').info()`:
Assume you are the author of a library or addon that relies on another library (e.g. `myAwesomeMaterialToolbar`). You can request that your addon is loaded after `ngMaterial`, so you can detect its version and do different things. E.g.:

```js
// my-awesome-material-toolbar.js
var mdVersion = angular.module('ngMaterial').info().version;
var deps = (mdVersion >= '1.1.1') ? [] : ['myShimForAFeatureAddedInNgMaterial1.1.1'];

angular.module('myAwesomeMaterialToolbar', deps);
```